wordfugue/sitesettings/models.py

```diff
@@ -4,7 +4,7 @@
 
 
 class SiteSettings(models.Model):
-    site = models.ForeignKey(Site)
+    site = models.ForeignKey(Site, on_delete=models.CASCADE)
```

Comment:
Here I'm setting the `on_delete` again, but I'm saying "If the `site` on the other end of the relationship was deleted, delete this model as well." Hence, cascade.